wen-restart/src/wen_restart.rs

```diff
@@ -722,7 +722,7 @@ pub(crate) fn aggregate_restart_heaviest_fork(
                 .total_active_stake = current_total_active_stake;
             progress_changed = true;
         }
-        if progress_changed {
+        if progress_changed || first_time_entering_loop {
```

Comment:
can we just initialize `progress_changed` to true and remove the `first_time_entering_loop` flag

Reply:
We can. It's a bit ugly because we need to hack progress_last_sent, but I guess it's worth one fewer variable.